onnxruntime/core/providers/rocm/math/softmax_ck.cuh

```diff
@@ -56,8 +56,8 @@ auto GetCKSoftmaxTypeStringAndOps() {
     auto invoker = impl->MakeInvokerPointer();
 
     auto ck_softmax_op = [impl = std::move(impl), invoker = std::move(invoker)](const SoftmaxParams<InputT, OutputT>* params) -> Status {
-      AccDataType alpha{1.0f};
-      AccDataType beta{0.0f};
+      double alpha{1.0f};
+      double beta{0.0f};
```

Comment:
Could we use float for alpha/beta? Usually that is type in accumulation.

Reply:
Impossible, it is ck's decision to use double for alpha and beta. Use float cause compiler error after ck update.